### DISCUSSION TITLE
Cypress creates 2 user sessions in 1 test

### DISCUSSION THREAD

**Discussion by oobertas** [cypress-io/cypress#28639] [Questions and Help]
Hello, I am facing the following problem. The application in test only allows one session per user. However, with every test cypress starts 2 sessions for some reason which does not allow user to login successfully. How can this be fixed? (Logging in with API or using cy.session() is not an option here), login has to be fully imitated if it was user. Is there any configuration that can disable it, or is it a bug in cypress?

**Comment by npugach** [comment]
Facing the same issue right now. 
Is there any solution?
